Shared/UI/Views/DisplayableImageView.swift

```diff
@@ -35,19 +35,36 @@ class DisplayableImageView: BaseView {
         didSet {
             self.displayableTask?.cancel()
 
-            let displayableRef = self.displayable
+            // There's no need to load the displayable if we're already displaying it.
+            let isAlreadyDisplayed = self.displayable?.isEqual(to: oldValue) ?? false
+            if isAlreadyDisplayed && self.state == .success {
+                return
+            }
```

Comment:
Fixes little loading flickers that happen when a cell is reconfigured with the same image.